[SYNTH-17309] Add a fargate with aws secret manager tab to the private location setup (#27131)
DataDog/documentation: content/en/synthetics/platform/private_locations/_index.md

{{% /tab %}}

{{% tab "Fargate with AWS Secret Manager" %}}

> Create a secret in AWS secret manager to store all or part of the previously generated private location configuration. Keep in mind that the `publicKey` cannot be kept as it is in the configuration file. The secret should look like this:

Reviewer: ```suggestion
Create a secret in AWS secret manager to store all or part of the previously generated private location configuration. Keep in mind that the `publicKey` cannot be kept as it is in the configuration file. For example:
```

```json
{
    "datadogApiKey": "...",
    "id": "...",
    "site": "...",
    "accessKey": "...",
    "secretAccessKey": "...",
    "privateKey": "...",
    "pem": "...",
    "fingerprint": "..."
}
```

Permissions are required to allow the task definition and the AWS Fargate instance to read from the secret manager. See [Specifying sensitive data using Secrets Manager secrets in Amazon ECS][25]for more information.

Create a Fargate task definition that matches the following example, replacing the values in the list of secrets with the ARN of the secret you created in the previous step. For example: `arn:aws:secretsmanager:<region>:<account-id>:secret:<secret_arn>:<secret_key>::`.

> If you didn't save all the configuration in the secret manager, you can still pass the value as hardcoded string arguments, you can check the [Fargate tab](./?tab=fargate#install-your-private-location) and do a mix.

Reviewer: ```suggestion
If you didn't save all the configuration in the secret manager, you can still pass the value as hardcoded string arguments. Review the [Fargate tab](#install-your-private-location) for more information.
```

Author: Hello,
For this link removing `./?tab=fargate` will not open the fargate tab, which is what I would like to have, so that readers can access to the fargate content in one click and not two (one for the link and another to open the fargate tab). Is it something that we don't want to have ?

Reviewer: Ah sorry I may have copied that in wrong! Yes you are correct please go ahead and change it back to have it link correctly if you haven't already , and I'll approve the PR :)
